package.json

```diff
@@ -4,12 +4,13 @@
   "description": "Google Labs repo for rapid prototyping",
   "type": "module",
   "scripts": {
-    "sync": "npm i && npm run clean:build && npm run build",
+    "clean": "git clean -fdx",
+    "sync": "npm run clean && npm install && npm run build",
```

Comment:
I'm not sure about the usefulness of `sync` or `refresh`. Generally you don't need to `npm run build` everything, since each package's build script knows how to build by itself now.

`clean` and `update` seem like good ones to have, though.

Reply:
```suggestion
```